bug: panic when translate ingress version

### Issue description

panic when translate extensions/v1beta1.Ingress

### Environment

* your apisix-ingress-controller version (output of `apisix-ingress-controller version --long`);

```
Version: 0.5.0
Git SHA: 98d0162
Go Version: go1.13.8
Building OS/Arch: linux/amd64
Running OS/Arch: linux/amd64
```

* your Kubernetes cluster version (output of `kubectl version`);

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.4", GitCommit:"c96aede7b5205121079932896c4ad89bb93260af", GitTreeState:"clean", BuildDate:"2020-06-17T11:41:22Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.4-tke.6", GitCommit:"194201819cf1e5cf45d38f72ce1aac9efca4c7ff", GitTreeState:"clean", BuildDate:"2020-12-29T09:13:24Z", GoVersion:"go1.15.6", Compiler:"gc", Platform:"linux/amd64"}
```

### Minimal test code / Steps to reproduce the issue

1. Use extensions/v1beta1.Ingress
2. set `config.kubernetes.ingressVersion=extensions/v1beta1`

### What's the actual result? (including assertion message & call stack if applicable)



```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x148 pc=0x15952ea]

goroutine 227 [running]:
github.com/apache/apisix-ingress-controller/pkg/kube/translation.(*translator).translateIngressV1beta1(0xc0004e0940, 0x0, 0x12, 0x182fd80, 0xc0003b0900, 0x17, 0xc000137450, 0x0, 0x0, 0xc000137450)
/build/pkg/kube/translation/ingress.go:94 +0x3a
github.com/apache/apisix-ingress-controller/pkg/kube/translation.(*translator).TranslateIngress(0xc0004e0940, 0x1c44c00, 0xc0003b0900, 0xc00062a3a8, 0x17, 0x1c44c00, 0xc0003b0900, 0x0, 0x0, 0xc0003829e0, ...)
/build/pkg/kube/translation/translator.go:198 +0x86
github.com/apache/apisix-ingress-controller/pkg/ingress.(*ingressController).sync(0xc000352680, 0x1c3c340, 0xc0003bc200, 0xc0005ee240, 0xc0004e5100, 0x4392e6)
/build/pkg/ingress/ingress.go:131 +0x2dc
github.com/apache/apisix-ingress-controller/pkg/ingress.(*ingressController).runWorker(0xc000352680, 0x1c3c340, 0xc0003bc200)
/build/pkg/ingress/ingress.go:80 +0x57
created by github.com/apache/apisix-ingress-controller/pkg/ingress.(*ingressController).run
/build/pkg/ingress/ingress.go:68 +0x1d9
```

### What's the expected result?

handle ingress correctly

--- a/pkg/kube/translation/translator.go
+++ b/pkg/kube/translation/translator.go
@@ -192,8 +192,14 @@ func (t *translator) TranslateUpstreamNodes(endpoints *corev1.Endpoints, port in
 }
 
 func (t *translator) TranslateIngress(ing kube.Ingress) ([]*apisixv1.Route, []*apisixv1.Upstream, error) {
-	if ing.GroupVersion() == kube.IngressV1 {
+	switch ing.GroupVersion() {
+	case kube.IngressV1:
 		return t.translateIngressV1(ing.V1())
+	case kube.IngressV1beta1:
+		return t.translateIngressV1beta1(ing.V1beta1())
+	case kube.IngressExtensionsV1beta1:
+		return t.translateIngressExtensionsV1beta1(ing.ExtensionsV1beta1())
+	default:
+		return nil, nil, fmt.Errorf("translator: source group version not supported: %s", ing.GroupVersion())
 	}
-	return t.translateIngressV1beta1(ing.V1beta1())
 }

